### DISCUSSION TITLE
The datepicker is reset to the current date when clear value

### DISCUSSION THREAD

**Discussion by chinnxz** [NG-ZORRO/ng-zorro-antd#8140] [Q&A]
Hi, I used the ng-zorro version 13 and  I want to disable this auto set to the current date when clear the date value, Can you have any ideas to do about this? thanks

**Comment by ParsaArvanehPA** [comment]
Hi, i'm checking date-picker component in version 13, <a  href="https://ng.ant.design/version/13.4.x/components/date-picker/en">here</a>. 
I pick a date, then press `x` mark to clear it, and it clears it. It is not setting the date to today's date.
If your problem still exists please provide more information.
